CONTRIBUTING.md

```diff
@@ -100,6 +105,14 @@ $ cargo version
 cargo 1.69.0 (6e9a83356 2023-04-12)
 ```
 
+#### Install Docker
+
+Currently, opendal uses Docker to set up environment for behaviour tests. Native Docker has some limitations, please check (https://github.com/apache/iceberg-rust/pull/748). Please use Orbstack.
```

Comment:
Hi, please don't cross link to other links in CONTRIBUTING guide. It's also not a good idea to redirect our developers to other porjects link. Please add them in our docs directly. 

Also, please add them under behavior tests since not all developers are required to install docker.

Reply:
ma ke